contracts/vectis-tests/tests/dao_chain/factory_govec_mgmt.rs

```diff
@@ -47,7 +55,7 @@ fn proxy_mint_govec_works() {
 }
 
 #[test]
-fn non_proxy_cannot_mint_on_govec() {
+fn cannot_mint_govec_without_paying_fee() {
```

Comment:
for this to be accurate, I think we need to make sure that the executor of the call has claimable govec, i.e. is a proxy wallet, l71 should be replaced with a wallet addr